templates/taxi/confirm_delete.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  {% if object is manufacturer %}
+    <h1>Delete {{ object.name }}?</h1>
+    <h4>
+      <strong>Notice, that all cars made by this manufacturer will be deleted!</strong>
+    </h4>
+
+  {% elif object is car %}

```

Comment:
Using `is` for object type checking might not work as expected in Django templates. Consider using `{% if object|instanceof:'Car' %}` with a custom template filter to check the object's type.